airflow/hooks/dbapi_hook.py

```diff
@@ -106,7 +106,7 @@ def get_sqlalchemy_engine(self, engine_kwargs=None):
             engine_kwargs = {}
         return create_engine(self.get_uri(), **engine_kwargs)
 
-    def get_pandas_df(self, sql, parameters=None):
+    def get_pandas_df(self, sql, parameters=None, **kwargs):
```

Comment:
Would you mind adding information to method docstring?

Reply:
Added